Add concept exercise functions (#1183)
exercism/javascript: exercises/concept/lasagna-master/.docs/instructions.md

# Instructions

In this exercise you are going to write some more code related to preparing and cooking your brilliant lasagna from your favorite cookbook.

You have five tasks.
The first one is related to the cooking itself, the other four are about the perfect preparation.

## 1. Determine whether the lasagna is done

When you have lasagna in the oven, you want to know whether you can already take it out or not.
To make sure the lasagna does not burn in the oven, you usually set a timer.
But sometimes you forget about that.

> Write a function `cookingStatus` with that accepts the remaining time on the timer in minutes as parameter.

Reviewer: ```suggestion
Write a function `cookingStatus` that accepts a single parameter, indicating the remaining time on the timer in minutes.
```

For each sauce layer in your lasagna, you will need 0.2 liters of sauce.

Define the function `quantities` that takes an array of layers as parameter.
The function will then determine the quantity of noodles and sauce needed to make your meal.
The result should be returned as an object with keys `noodles` and `sauce`.

```javascript
quantities(['sauce', 'noodles', 'sauce', 'meat', 'mozzarella', 'noodles']);
// => { noodles: 100, sauce: 0.4 }
```

## 4. Add the secret ingredient

A while ago you visited a friend and ate lasagna there.
It was amazing and had something special to it.
The friend sent you the list of ingredients and told you the last item on the list is the "secret ingredient" that made the meal so special.
Now you want to add that secret ingredient to your recipe as well.

Write a function `addSecretIngredient` that accepts two arrays of ingredients as parameters.
The first parameter is the list your friend sent you, the second is the ingredient list for your own recipe.
The function should add the last item from your friends list to the end of your list.
The array that represents your recipe should be modified directly and the function should not return anything.

> However, the first argument should not be modified.

Reviewer: ```suggestion
However, your friend's list should not be modified.
```
